### PR TITLE
Version 2: Add "Try again" button for Connection Not Detected Dialog

### PR DESCRIPTION
**Pull request by OleksandrSPV**
Added "Try again" button for Connection Not Detected Dialog

https://github.com/user-attachments/assets/c390d266-a104-4e23-8103-9951b107d2c9

Changed handle "Try again" for Connection Not Approved Dialog

https://github.com/user-attachments/assets/cd9171af-f044-4751-a9e8-05d41ed8bb31

### CODE REVIEW

File: src/components/connection-not-detected/ConnectionNotDetected.tsx [hunk 1]
```diff
@@ -29,6 +29,15 @@ export const ConnectionNotDetected: React.FunctionComponent = () => {
         >
           {t("connection.error.connection-not-detect.button")}
         </Button>
+        <Button
+          variant="contained"
+          color="primary"
+          onClick={() => {
+            window.location.reload();
```
Comment (donskov) [inline src/components/connection-not-detected/ConnectionNotDetected.tsx:36]: Is there any reason to window reload instead of calling our start flow?

Comment (OleksandrSPV) [reply src/components/connection-not-detected/ConnectionNotDetected.tsx]: @donskov Is this way ok for you? https://github.com/PeculiarVentures/fortify-tools/pull/196/commits/62c30b4b4cba1cbcf4fb92c8a8b085502ea9d735

Comment (donskov) [reply src/components/connection-not-detected/ConnectionNotDetected.tsx]: @OleksandrSPV I don't really understand why we need another state just to make a new connection. Can we call the `start` method instead?

Comment (OleksandrSPV) [reply src/components/connection-not-detected/ConnectionNotDetected.tsx]: Unfortunately, it doesn't work with the same `FortifyAPI` instance for me. I have an error when `challenge` method called in `tryLogin`: `Socket connection is not open at Client.checkSocketState`
Do you have any ideas? @donskov

Comment (donskov) [reply src/components/connection-not-detected/ConnectionNotDetected.tsx]: @OleksandrSPV Yeah, I see. Let's back to the `window.reload` until we find a better solution.

Comment (OleksandrSPV) [reply src/components/connection-not-detected/ConnectionNotDetected.tsx]: @donskov Ok, done, thanks https://github.com/PeculiarVentures/fortify-tools/pull/196/commits/63f46030cd4dd43075877f897d310bd7f6e1795b

**Review by donskov — APPROVED** [review]

**Pull request merged**